### ISSUE TITLE
Inaccurate Info, please update this article

### ISSUE THREAD

**Issue by musashiharuno** [MicrosoftDocs/OfficeDocs-Support#232]
[Enter feedback here]

the below info is not accurate. Please update the article.

Per validation, as long as the user has "Get-RemoteDomain" RBAC permisison, that it will redirect to O365 Login Page, for those without this particular permission, it turns to O365 Product Page.

 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b4c0bb69-5903-6063-c5a2-b7017fbf636c
* Version Independent ID: 9f452bfc-0dfa-9807-5c14-423b3f7b7374
* Content: [Office 365 link in on-premises EAC goes to product comparison - Exchange](https://docs.microsoft.com/en-us/exchange/troubleshoot/admin-center/o365-link-eac-redirect-product-comparison)
* Content Source: [Exchange/ExchangeServer/admin-center/o365-link-eac-redirect-product-comparison.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Exchange/ExchangeServer/admin-center/o365-link-eac-redirect-product-comparison.md)
* Service: **o365-administration**
* GitHub Login: @simonxjx
* Microsoft Alias: **v-six**

**Comment by nam31** [comment]
@musashiharuno, this article has been updated recently. Please let us know if your issue is resolved now. Thanks

**Comment by musashiharuno** [comment]
@nam31 , it appears that the article remains the same info quoted at https://docs.microsoft.com/en-us/exchange/troubleshoot/administration/o365-link-eac-redirect-product-comparison . Any pages implies the article updated recently since 2019.

**Comment by MaratMussabekov** [comment]
Hello @musashiharuno ,
there is no such RBAC permisison in Exchange Online, could you please advise, which one from [the list](https://docs.microsoft.com/en-us/exchange/permissions-exo/permissions-exo) did you mean, so I can reproduce it and check?
Thank you

**Comment by musashiharuno** [comment]
@MaratMussabekov 
the docs https://docs.microsoft.com/en-us/exchange/troubleshoot/administration/o365-link-eac-redirect-product-comparison is regarding the behavior when user sign in onPrem Exchange ECP, where on the top of the left navigation button there are 2 Items -> Exchange / Office365. By clicking the button Office365 from there, it's reported that end user requires "View-Only Organization Management role group or the Organization Management role group" so that it won't redirect them to Office365 product page. 

Testing and dig dive indicates as long as the user has the Get-RemoteDomain permission (as the minimal permission rather than org level RBAC ), it can also address the behavior. 

In other words, onPrem Exchange Admin need to implement custom RBAC to allow end user has the ability to query onPrem remote domain configuration

**Comment by MaratMussabekov** [comment]
Hello @musashiharuno ,
I reproduced the scenario:
1. created a custom role that only contains "Get-RemoteDomain" role entry
2. assigned it to a user
3. logged in to ECP, only able to see the "accepted domains" tab of "mail flow" section
4. wait one hour so everything could synchronize to the cloud
5. clicked on Office 365 button, and still was redirected to Compare All Microsoft 365 Products page...

Could you please check one more time? and make sure that no other permission was assigned to the test user.

Thank you

**Comment by nam31** [comment]
@musashiharuno Please let us know if you still have any question regarding the issue you have raised initially.
The issue is being closed since your question seems to be answered. Feel free to re-open this issue if you feel that it hasn't been answered or that there are further suggestions to improve the documentation itself.
Thanks.
